python/pylibcugraph/pyproject.toml

```diff
@@ -26,8 +27,10 @@ authors = [
 license = { text = "Apache 2.0" }
 requires-python = ">=3.9"
 dependencies = [
-    "pylibraft==24.8.*",
-    "rmm==24.8.*",
+    "cupy-cuda11x>=12.0.0",
+    "libcugraph==24.6.*",
+    "pylibraft==24.6.*",
+    "rmm==24.6.*",
```

Comment:
hey @msarahan I know this is still a draft and not ready for review, but was just skimming it and wanted to note that I noticed this... looks like you still have a bunch of `24.6` dependencies in here, even though this is targeting release 24.08. Those should all get updated.

```shell
ci/release/update-version.sh '24.08.00'
```

Reply:
I think that should be changed in dependencies.yaml, and then this pyproject.toml should be regenerated. At any rate, thanks. I have only just merged in branch-24.08, and will start seeing what other fixes are necessary.